build-monitor-plugin/src/main/java/com/smartcodeltd/jenkinsci/plugins/buildmonitor/BuildMonitorView.java

```diff
@@ -143,11 +161,28 @@ private boolean isGiven(String value) {
 
     private List<JobView> jobViews() {
         JobViews views = new JobViews(new StaticJenkinsAPIs(), currentConfig());
+        List<JobView> jobs = new ArrayList<JobView>();
 
         //A little bit of evil to make the type system happy.
         @SuppressWarnings("unchecked")
+        List<ItemGroup<?>> groups = new ArrayList(filter(super.getItems(), AbstractFolder.class));
```

Comment:
What happens if the Folder plugin is not installed and the class is not available?